include/lsst/jointcal/ConstrainedPhotometryModel.h

```diff
@@ -118,6 +118,17 @@ class ConstrainedPhotometryModel : public PhotometryModel {
     /// Return the initial calibration to use from this photoCalib.
     virtual double initialChipCalibration(std::shared_ptr<afw::image::PhotoCalib const> photoCalib) = 0;
 
+    /// To hold the return of prepPhotoCalib
+    struct PrepPhotoCalib_t {
```

Comment:
The `_t` suffix here is a pattern that one sees in e.g. the C++ standard library to disambiguate between things and types-of-things.  That's not usually a problem for us because we capitalize types.  When we need to disambiguate between two types (i.e. a class and a template parameter) our conventions say to use a capital `T` suffix, but I *think* even that's not needed here.